requirements.txt

```
@@ -1,130 +1,136 @@
-alabaster~=0.7.12
+# this file contains constrants and not requirements for our CI job
+# it is not intended to be used outside CI but we keep the name requirements.txt
+# since some tools are hard coded to look for that name when scanning requirements
+# This should contain all post 1.0.0 packages constrained to a minor version e.g. with ~=
+# and all pre 1.0.0 pacakges and packages not using a
+# 3 part version number to an exact version using ==
+alabaster==0.7.12
 argon2-cffi~=21.3.0
 argon2-cffi-bindings~=21.2.0
 asttokens~=2.1.0
 attrs~=22.1.0
-autodocsumm~=0.2.9
+autodocsumm==0.2.9
 azure-core~=1.26.0
 azure-identity~=1.12.0
 Babel~=2.11.0
-backcall~=0.2.0
+backcall==0.2.0
 beautifulsoup4~=4.11.1
 bleach~=5.0.1
-broadbean~=0.11.0
+broadbean==0.11.0
 cachetools~=5.2.0
 certifi~=2022.9.24
 cffi~=1.15.1
 charset-normalizer~=2.1.1
 click~=8.1.3
 click-default-group~=1.2.2
-colorama~=0.4.5
+colorama-=0.4.5
```

typo? or is minus also allowed?
```suggestion
colorama==0.4.5
```

---

This was a typo (fixed elsewhere)